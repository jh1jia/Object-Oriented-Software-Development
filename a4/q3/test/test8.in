stdin dropfirst 2 count l doublewords count 2 count 1 count 3 allcaps
HEllo WorLd
