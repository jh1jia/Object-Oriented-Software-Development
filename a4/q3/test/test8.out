12O
12O
RLD
RLD
